### PR TITLE
automated code cleanup

### PR DESCRIPTION
**Pull request by ivansenic**
**What this PR does**:
Christmas code maintenance (part two): Run automated code cleanup from the IDE. Mainly cleanup, final assignments, unnecessary typing, `toString()` calls, etc. 

Although it's automated action, please review with care.. I am open for any suggestions for reverting..

### CODE REVIEW

File: persistence-api/src/main/java/io/stargate/db/schema/SchemaBuilderImpl.java [hunk 1]
```diff
@@ -51,27 +51,27 @@
...
   private boolean indexEntries;
   private boolean indexFull;
   private String indexClass;
   private Map<String, String> indexOptions = new HashMap<>();
-  private List<Column> fromColumns = new ArrayList<>();
-  private List<Column> toColumns = new ArrayList<>();
+  private final List<Column> fromColumns = new ArrayList<>();
+  private final List<Column> toColumns = new ArrayList<>();
   private Map<String, String> replication = Collections.emptyMap();
   private Optional<Boolean> durableWrites = Optional.empty();
```
Comment (ivansenic) [inline persistence-api/src/main/java/io/stargate/db/schema/SchemaBuilderImpl.java:76]: this seems mixed now, maybe revert?

File: graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/processor/ProcessingLogType.java [hunk 2]
```diff
@@ -20,5 +20,4 @@
 public enum ProcessingLogType implements ErrorClassification {
   Info,
   Warning,
-  ;
```
Comment (mpenick) [inline graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/processor/ProcessingLogType.java:23]: There was a lot of these trailing semicolons. I wonder why?

**Review by mpenick — APPROVED** [review]

File: graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/processor/ProcessingLogType.java (hunk 2 above)
Comment (ivansenic) [reply graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/processor/ProcessingLogType.java:23]: No idea..

**Review by dougwettlaufer — APPROVED** [review]

File: persistence-api/src/main/java/io/stargate/db/schema/SchemaBuilderImpl.java (hunk 1 above)
Comment (dougwettlaufer) [reply persistence-api/src/main/java/io/stargate/db/schema/SchemaBuilderImpl.java:76]: Or reorganize so the `final' are together?

Comment (ivansenic) [reply persistence-api/src/main/java/io/stargate/db/schema/SchemaBuilderImpl.java]: Reverted.

**Pull request merged**